front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/CraftComponents/ThreeColumn/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+// components
+import { Box, useBreakpoint } from '@citizenlab/cl2-component-library';
+
+// craft
+import { Element, ROOT_NODE, useNode } from '@craftjs/core';
+
+// i18n
+import messages from './messages';
+import Container from '../Container';
+
+const ThreeColumn = () => {
```

Comment:
The column elements are also duplicated to make sure we don't affect the other content builders with this padding. As we converge towards the new system, we will remove this duplicate code.